### DISCUSSION TITLE
3.0.0-beta.2 - Material UI 5.0.0 Integration

### DISCUSSION THREAD

**Discussion by vip-git** [vip-git/react-jsonschema-form-material-ui#409] [Releases]
# This release marks full support for MUI 5

- From this release onwards we will be supporting [MUI 5](https://github.com/mui-org/material-ui/releases/tag/v5.0.0) as released by [Material UI](https://mui.com/blog/mui-core-v5/)
- This release also marks the very first beta version with [universal schema integration](https://github.com/vip-git/react-jsonschema-form-material-ui/blob/main/universal-json-schema.md) - In future releases more information on this will be given in detail on its use.
- From this release onwards you can leverage `theme` property when initialising the form to provide [Material UI Theme object](https://mui.com/customization/color/#playground) .
 
```
npm install --save react-jsonschema-form-material-ui@3.0.0-beta.2
```

**Demo URL:** 
- https://react-jsonschema-form-material-ui-github56.vercel.app

**CRA Example URL:**
- https://cra-react-jsonschema-form-material-ui-example.vercel.app

**NextJS Example URL:**
- https://nextjs-react-jsonschema-form-material-ui-example.vercel.app/

### Previously:
- This minor release adds support for [Material UI Autocomplete](https://material-ui.com/api/autocomplete/) Component.
- Some bug fixes on array schema types.

**Next Releases**
- Next release onwards we will move to `beta.x` followed by `rc.x` to finalize `3.0.0`
- Support for [Material UI 5 - Autocomplete](https://next.material-ui.com/components/autocomplete/) will be included in future pre-releases tagged `-mui-5` similar previous [example release](https://github.com/vip-git/react-jsonschema-form-material-ui/releases/tag/3.0.0-alpha.6-mui-5).

Demo URL:  https://react-jsonschema-form-material-ui.github56.now.sh

### Example Usage:
- Form Schema
```
{
  "title": "Auto Complete example",
  "description": "Example demo for Material UI Autocomplete component usage",
  "type": "object",
  "properties": {
    "auto-complete": {
        "type": "string", // For multi-select this can be "array"
        "title": "Example Auto Complete",
        "enum": [
            "Yes",
            "No"
        ]
    }
  }
}
```
- UI Schema
```
{
    "auto-complete": {
        "ui:widget": "material-auto-complete",
        "ui:props": { // Available props support come from https://material-ui.com/api/autocomplete/#props
            "disableClearable": false
        }
    }
}
```

- Form Data
```
{}
```

<hr /><em>This discussion was created from the release <a href='https://github.com/vip-git/react-jsonschema-form-material-ui/releases/tag/3.0.0-beta.2'>3.0.0-beta.2 - Material UI 5.0.0 Integration</a>.</em>

**Comment by msueping** [comment]
Hi. I've been trying to get the Tabs example to work but cannot. I'm able to show the tabs and the contents of the first tab only. If I click on any other tab, nothing happens. I see no errors in the console either.

I am using your latest version (3.0.0. beta 2) along with the latest Material UI v5.

I am able to copy all of the other example schemas shown in the showcase and get them to work in my code. It is only the two examples that use the tabs that don't do anything when I click on a tab other than the first one. Do you have any idea why that would be happening?

Is there any chance that you'd be able to put these examples on codepen or codesandbox?

BTW, here's a screen shot showing how it looks in my test project. When I click "Change Data" nothing happens. It's rendered in a dialog and I'm not sure if that has anything to do with it. 
![image](https://user-images.githubusercontent.com/67658122/134422547-183648ab-3e2a-4d73-aad8-9b0e24f0dfae.png)

**Reply by vip-git** [reply]
Hi @msueping - Sorry for the delayed reply. Would have helped if it was created as a issue .

Are you able to share your `ui-schema.json` , `schema.json` for this UI as well ?

Thanks - V
